frontend/institution/allInstitutionsController.js

```diff
@@ -80,7 +80,7 @@
             .then(function success(response) {
                 currentPage += 1;
                 moreInstitutions = response.next;
-
+                allInstitutionsCtrl._checkSeenInstitutions(response.institutions);
```

Comment:
shouldnt setInstitution depend on this?

Reply:
I didn't get it, what u mean by 'depend'?